### PR TITLE
Fix link to RAPIDS cudf in index.md

### PR DESCRIPTION
**Pull request by sameerz**
1. Please write a description in this text box of the changes that are being
   made.

Fixed link to RAPIDS cudf in index.md

2. Please ensure that you have written units tests for the changes made/features
   added.

Tested on Jekyll locally.

### CODE REVIEW

**Comment by sameerz** [comment]
build

**Comment by revans2** [comment]
precommit build is not working for branch-0.2 yet

File: docs/index.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ description: This site serves as a collection of documentation about the RAPIDS
 ---
 As data scientists shift from using traditional analytics to leveraging AI applications that better model complex market demands, traditional CPU-based processing can no longer keep up without compromising either speed or cost. The growing adoption of AI in analytics has created the need for a new framework to process data quickly and cost efficiently with GPUs.
 
-The RAPIDS Accelerator for Apache Spark combines the power of the <a href="github.com/rapidsai/cudf/">RAPIDS cuDF</a> library and the scale of the Spark distributed computing framework.  The RAPIDS Accelerator library also has a built-in accelerated shuffle based on <a href="https://github.com/openucx/ucx/">UCX</a> that can be configured to leverage GPU-to-GPU communication and RDMA capabilities. 
+The RAPIDS Accelerator for Apache Spark combines the power of the <a href="http://github.com/rapidsai/cudf/">RAPIDS cuDF</a> library and the scale of the Spark distributed computing framework.  The RAPIDS Accelerator library also has a built-in accelerated shuffle based on <a href="https://github.com/openucx/ucx/">UCX</a> that can be configured to leverage GPU-to-GPU communication and RDMA capabilities.
```
Comment (abellina) [inline docs/index.md:10]: I'd make this `https://`, it seems like we normally use https in our other links.

**Comment by jlowe** [comment]
> build

Unfortunately CI isn't connected on branch-0.2 yet, so I don't think we're going to get a CI response.

File: docs/index.md (hunk 1 above)
Comment (abellina) [inline docs/index.md:10]:
```suggestion
The RAPIDS Accelerator for Apache Spark combines the power of the <a href="https://github.com/rapidsai/cudf/">RAPIDS cuDF</a> library and the scale of the Spark distributed computing framework.  The RAPIDS Accelerator library also has a built-in accelerated shuffle based on <a href="https://github.com/openucx/ucx/">UCX</a> that can be configured to leverage GPU-to-GPU communication and RDMA capabilities. 
```

Comment (abellina) [reply docs/index.md:10]: nit, the http one does redirect to https, but we use https in another link in this page.

Comment (sameerz) [reply docs/index.md]: Good catch, updated to https

**Comment by abellina** [comment]
So I think this is going to get into branch-0.2, but the docs not updated. We may want to back-port it to branch-0.1/master?

**Comment by revans2** [comment]
@sameerz Do we want to wait for the new build pipeline to be setup or should I merge this in as is?

**Comment by sameerz** [comment]
Please merge as is.  This is only a doc change, no code change.  And once merged, please push to master as well.  We will continue working on the docs in branch-0.2 going forward.

**Pull request merged**